<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogPathologie</class>
 <widget class="QDialog" name="DialogPathologie">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Formulaire de Pathologie</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/newPrefix/images/icone.png</normaloff>:/newPrefix/images/icone.png</iconset>
  </property>
  <property name="toolTip">
   <string>Documents relatifs cette pathologie</string>
  </property>
  <widget class="QPushButton" name="pushButton_PlanTherapeutique">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>80</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>27</height>
    </size>
   </property>
   <property name="text">
    <string>Plan Thérapeutique</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>11</y>
     <width>561</width>
     <height>106</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>81</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Domaine :</string>
        </property>
        <property name="buddy">
         <cstring>comboBox_PathologieDomaine</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MyComboBox" name="comboBox_PathologieDomaine">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>27</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">selection-background-color: rgb(255, 255, 127);</string>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_DomaineSelection">
        <property name="toolTip">
         <string>Ajouter un domaine pour la pathologie en cours</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/newPrefix/images/valid.png</normaloff>:/newPrefix/images/valid.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_AddSelection_2">
        <property name="toolTip">
         <string>Créer un nouveau domaine</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/newPrefix/images/add1.png</normaloff>:/newPrefix/images/add1.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_DeleteDommaine">
        <property name="toolTip">
         <string>Supprimer ce domaine</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/newPrefix/images/delete2.png</normaloff>:/newPrefix/images/delete2.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="maximumSize">
         <size>
          <width>81</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Pathologie :</string>
        </property>
        <property name="buddy">
         <cstring>comboBox_PathologiePathologie</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MyComboBox" name="comboBox_PathologiePathologie">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>27</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Pour ajouter une nouvelle pathologie, tapez sont nom ici.</string>
        </property>
        <property name="styleSheet">
         <string notr="true">selection-background-color: rgb(255, 255, 127);</string>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_AddPathologie">
        <property name="toolTip">
         <string>Créer une pathologie</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/newPrefix/images/add1.png</normaloff>:/newPrefix/images/add1.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_DeletePathologie">
        <property name="toolTip">
         <string>Supprimer cette pathologie</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/newPrefix/images/delete2.png</normaloff>:/newPrefix/images/delete2.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="maximumSize">
         <size>
          <width>81</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Sélection :</string>
        </property>
        <property name="buddy">
         <cstring>comboBox_PathologieSelection</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MyComboBox" name="comboBox_PathologieSelection">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>27</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">selection-background-color: rgb(255, 255, 127);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_AddSelection">
        <property name="minimumSize">
         <size>
          <width>27</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>27</width>
          <height>27</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Ajouter la pathologie en cours dans la selection</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/newPrefix/images/add1.png</normaloff>:/newPrefix/images/add1.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_deleteSelection">
        <property name="maximumSize">
         <size>
          <width>27</width>
          <height>27</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Supprimer cette pathologie de la selection</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/newPrefix/images/delete2.png</normaloff>:/newPrefix/images/delete2.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_SavePathologie">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>30</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Enregistrer Pathologie</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>131</y>
     <width>765</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>145</y>
     <width>762</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineEdit_NomReference">
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>Modifier le nom de référence ici et Entrée pour valider</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_Chronic">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>Maladie chronique</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_documentPathologie">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/newPrefix/images/doc_all.png</normaloff>:/newPrefix/images/doc_all.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_traitements">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Traitements proposés</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/newPrefix/images/gelule.png</normaloff>:/newPrefix/images/gelule.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Descriptif :</string>
          </property>
          <property name="buddy">
           <cstring>plainTextEdit_Descriptif</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit_Descriptif">
          <property name="minimumSize">
           <size>
            <width>280</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Domaines :</string>
          </property>
          <property name="buddy">
           <cstring>listWidget_Synonymes</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listWidget_Domaines">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>220</width>
            <height>150</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Double Click oiur supprimer un domaine de la liste</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Synonymes :</string>
          </property>
          <property name="buddy">
           <cstring>listWidget_Synonymes</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listWidget_Synonymes">
          <property name="maximumSize">
           <size>
            <width>240</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Examen :</string>
        </property>
        <property name="buddy">
         <cstring>comboBox_Examen</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MyComboBox" name="comboBox_Examen">
        <property name="minimumSize">
         <size>
          <width>230</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>27</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">selection-background-color: rgb(255, 255, 127);</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>18</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Critère :</string>
        </property>
        <property name="buddy">
         <cstring>comboBox_Critere</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MyComboBox" name="comboBox_Critere">
        <property name="minimumSize">
         <size>
          <width>320</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>27</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">selection-background-color: rgb(255, 255, 127);</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_AddCritere">
        <property name="toolTip">
         <string>Editer les critères diagnostiques</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/newPrefix/images/edit1.png</normaloff>:/newPrefix/images/edit1.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="MyTableWidget" name="tableWidget_Criteres">
        <property name="rowCount">
         <number>10</number>
        </property>
        <property name="columnCount">
         <number>6</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderVisible">
         <bool>true</bool>
        </attribute>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <column/>
        <column/>
        <column/>
        <column/>
        <column/>
        <column/>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <property name="spacing">
       <number>20</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>448</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_Cancel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Annuler</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_Save">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Enregistrer</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyComboBox</class>
   <extends>QComboBox</extends>
   <header>Mywidgets</header>
  </customwidget>
  <customwidget>
   <class>MyTableWidget</class>
   <extends>QTableWidget</extends>
   <header>Mywidgets</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
